mlflow/store/file_store.py

```diff
@@ -59,9 +59,11 @@ def __init__(self, root_directory=None, artifact_root_uri=None):
         Create a new FileStore with the given root directory and a given default artifact root URI.
         """
         super(FileStore, self).__init__()
-        self.root_directory = root_directory or _default_root_dir()
-        self.artifact_root_uri = artifact_root_uri or self.root_directory
-        self.trash_folder = build_path(self.root_directory, FileStore.TRASH_FOLDER_NAME)
+        root_directory = root_directory or _default_root_dir()
+        if artifact_root_uri is not None:
+            self.artifact_root_uri = artifact_root_uri
```

Comment:
will change to convention of
self.artifact_root_uri = artifact_root_uri if artifact_root_uri is not None \
                                    else local_uri_from_path(parse_path(root_directory))

# waiting for builds first